app/presenters/corporate_information_page_presenter.rb

```diff
@@ -0,0 +1,157 @@
+class CorporateInformationPagePresenter < ContentItemPresenter
+  def initialize(content_item, view_context)
+    super(content_item)
+    @view_context = view_context
+  end
+
+  def page_title
+    page_title = super
+    page_title += " - #{default_organisation['title']}" if default_organisation
+    page_title
+  end
+
+  def organisation_logo(organisation = default_organisation)
+    return nil unless organisation && organisation.dig("details", "logo")
+
+    logo = organisation["details"]["logo"]
+    logo_component_params = {
+      organisation: {
+        name: logo["formatted_title"].html_safe,
+        url: organisation["base_path"],
+        brand: organisation_brand(organisation),
+        crest: logo["crest"],
+      },
+    }
+
+    if logo["image"]
+      logo_component_params[:organisation][:image] = {
+        url: logo["image"]["url"],
+        alt_text: logo["image"]["alt_text"],
+      }
+    end
+
+    logo_component_params
+  end
+
+  def organisation_brand_class(organisation = default_organisation)
+    organisation_brand(organisation).present? ? "#{organisation_brand(organisation)}-brand-colour" : nil
+  end
+
+  def corporate_information?
+    corporate_information_groups.any?
+  end
+
+  def corporate_information
+    corporate_information_groups.map do |group|
+      {
+        heading: @view_context.tag.h3(
+          group["name"],
+          id: group_title_id(group["name"]),
+        ),
+        links: normalised_group_links(group),
+      }
+    end
+  end
+
+  def corporate_information_heading_tag
+    @view_context.tag.h2(
+      corporate_information_heading[:text],
+      id: corporate_information_heading[:id],
+    )
+  end
+
+  def further_information
+    [
+      further_information_about("publication_scheme"),
+      further_information_about("welsh_language_scheme"),
+      further_information_about("personal_information_charter"),
+      further_information_about("social_media_use"),
+      further_information_about("about_our_services"),
+    ].join(" ").html_safe
+  end
+
+private
+
+  def default_organisation
+    organisation_content_id = content_item.content_store_hash.dig("details", "organisation")
+    organisation = nil
+    if organisations.present?
+      organisation = organisations.detect { |org| org["content_id"] == organisation_content_id }
+      raise "No organisation in links that matches the one specified in details: #{organisation_content_id}" unless organisation
```

Comment:
We don't test this raise. Is it possible to get here at all with our current content?